tests/specs/task/package_json_node_modules_dir_manual/deno.json

```diff
@@ -0,0 +1,3 @@
+{
+  "nodeModulesDir": "manual"
```

Comment:
> When migrating the nodeModulesDir: false, i received a warning to use nodeModulesDir: none instead but the expected behaviour in the test didn't match that is I had to switch to nodeModulesDir: manual to get the expected behaviour.

I think this might be a bug. I don't think it should be auto-installing from the package.json in this case.

Reply:
I updated the option to "none" and updated the output to include the download messages for packages from package.json. 

The confusing part is why deno would install the packages even though they're not required to execute `deno task echo`

Reply:
Yeah, I think that's a bug.

Reply:
Investigated and it's correct that it's installing npm packages before running the tests because it needs to resolve the binary commands.